internals/overlord/servstate/manager_test.go

```diff
@@ -2093,3 +2094,49 @@ func waitChecks(c *C, checkMgr *checkstate.CheckManager, f func(checks []*checks
 	c.Fatal("timed out waiting for checks to settle")
 	return nil
 }
+
+func (s *S) TestMetrics(c *C) {
+	s.newServiceManager(c)
+	s.planAddLayer(c, testPlanLayer)
+	s.planChanged(c)
+
+	s.startTestServices(c, true)
+	if c.Failed() {
+		return
+	}
+	buf := new(bytes.Buffer)
+	openTelemetryWriter := metrics.NewOpenTelemetryWriter(buf)
+	s.manager.WriteMetrics(openTelemetryWriter)
+	metrics := buf.String()
+	c.Assert(metrics, testutil.Contains, "pebble_service_start_count{service=test1} 1")
```

Comment:
Should we just assert on the entire output, for simplicity, and so we're testing the HELP and TYPE parts?